header-only/layout-builder/awkward/LayoutBuilder.h

```diff
@@ -1201,22 +1179,6 @@ namespace awkward {
         return index_.length();
       }
 
-      /// @brief Checks for validity and consistency.
-      bool
-      is_valid(std::string& error) const noexcept {
-        if (content_.length() != last_valid_ + 1) {
-          std::stringstream out;
-          out << "Indexed node" << id_ << " has content length "
-              << content_.length() << " but last valid index is " << last_valid_
-              << "\n";
-          error.append(out.str());
-
-          return false;
-        } else {
-          return content_.is_valid(error);
-        }
-      }
```

Comment:
We still need an `is_valid` method. We can compare the `content_.length()` and `index_.length()` for this case.